java/libphonenumber/src/com/google/i18n/phonenumbers/MetadataLoader.java

```diff
@@ -23,7 +23,8 @@
  */
 public interface MetadataLoader {
   /**
-   * Returns an input stream corresponding to the metadata to load.
+   * Returns an input stream corresponding to the metadata to load. This method may be invoked from
+   * several threads, so implementations should be stateless.
```

Comment:
nit: I'd say.
This method may be called concurrently so implementations must be thread-safe.
(this is just more common nomenclature)